2023-11-23 12:56:31,850 INFO    AsyncIO Thread: default:129470 [wandb_setup.py:_flush():76] Current SDK version is 0.15.12
2023-11-23 12:56:31,850 INFO    AsyncIO Thread: default:129470 [wandb_setup.py:_flush():76] Configure stats pid to 129470
2023-11-23 12:56:31,850 INFO    AsyncIO Thread: default:129470 [wandb_setup.py:_flush():76] Loading settings from /home/ubuntu/.config/wandb/settings
2023-11-23 12:56:31,850 INFO    AsyncIO Thread: default:129470 [wandb_setup.py:_flush():76] Loading settings from /home/ubuntu/falcon/gti_repo/LLM-as-a-Service/Ray_demo/llmAPI/wandb/settings
2023-11-23 12:56:31,850 INFO    AsyncIO Thread: default:129470 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-11-23 12:56:31,850 WARNING AsyncIO Thread: default:129470 [wandb_setup.py:_flush():76] Could not save program above cwd: /home/ubuntu/dev/lib/python3.10/site-packages/ray/_private/workers/default_worker.py
2023-11-23 12:56:31,850 INFO    AsyncIO Thread: default:129470 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': None, 'program_abspath': '/home/ubuntu/dev/lib/python3.10/site-packages/ray/_private/workers/default_worker.py', 'program': '/home/ubuntu/dev/lib/python3.10/site-packages/ray/_private/workers/default_worker.py'}
2023-11-23 12:56:31,850 INFO    AsyncIO Thread: default:129470 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2023-11-23 12:56:31,850 INFO    AsyncIO Thread: default:129470 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2023-11-23 12:56:31,850 INFO    AsyncIO Thread: default:129470 [wandb_init.py:_log_setup():528] Logging user logs to /home/ubuntu/falcon/gti_repo/LLM-as-a-Service/Ray_demo/llmAPI/wandb/run-20231123_125631-q6e8ksr2/logs/debug.log
2023-11-23 12:56:31,851 INFO    AsyncIO Thread: default:129470 [wandb_init.py:_log_setup():529] Logging internal logs to /home/ubuntu/falcon/gti_repo/LLM-as-a-Service/Ray_demo/llmAPI/wandb/run-20231123_125631-q6e8ksr2/logs/debug-internal.log
2023-11-23 12:56:31,851 INFO    AsyncIO Thread: default:129470 [wandb_init.py:init():568] calling init triggers
2023-11-23 12:56:31,851 INFO    AsyncIO Thread: default:129470 [wandb_init.py:init():575] wandb.init called with sweep_config: {}
config: {}
2023-11-23 12:56:31,851 INFO    AsyncIO Thread: default:129470 [wandb_init.py:init():618] starting backend
2023-11-23 12:56:31,851 INFO    AsyncIO Thread: default:129470 [wandb_init.py:init():622] setting up manager
2023-11-23 12:56:31,853 INFO    AsyncIO Thread: default:129470 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-11-23 12:56:31,854 INFO    AsyncIO Thread: default:129470 [wandb_init.py:init():628] backend started and connected
2023-11-23 12:56:31,856 INFO    AsyncIO Thread: default:129470 [wandb_init.py:init():720] updated telemetry
2023-11-23 12:56:31,864 INFO    AsyncIO Thread: default:129470 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2023-11-23 12:56:32,269 INFO    AsyncIO Thread: default:129470 [wandb_run.py:_on_init():2220] communicating current version
2023-11-23 12:56:32,363 INFO    AsyncIO Thread: default:129470 [wandb_run.py:_on_init():2229] got version response upgrade_message: "wandb version 0.16.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-11-23 12:56:32,363 INFO    AsyncIO Thread: default:129470 [wandb_init.py:init():804] starting run threads in backend
2023-11-23 12:56:32,420 INFO    AsyncIO Thread: default:129470 [wandb_run.py:_console_start():2199] atexit reg
2023-11-23 12:56:32,420 INFO    AsyncIO Thread: default:129470 [wandb_run.py:_redirect():2054] redirect: wrap_raw
2023-11-23 12:56:32,420 INFO    AsyncIO Thread: default:129470 [wandb_run.py:_redirect():2119] Wrapping output streams.
2023-11-23 12:56:32,421 INFO    AsyncIO Thread: default:129470 [wandb_run.py:_redirect():2144] Redirects installed.
2023-11-23 12:56:32,422 INFO    AsyncIO Thread: default:129470 [wandb_init.py:init():845] run started, returning control to user process
